packages/message/src/PDRs.ts

```diff
@@ -71,3 +87,83 @@ export const getMessagePdrPANMessage = (
 export const getMessagePdrName = (
   message: MessageWithOptionalPayloadPdr
 ): string | undefined => getMessagePdr(message)?.name;
+
+/**
+ * Get the number of running executions for a PDR, if any.
+ *
+ * @param {MessageWithOptionalPdrStats} message - A workflow message
+ * @returns {number} Number of running executions
+ *
+ * @alias module:PDRs
+ */
+export const getMessagePdrRunningExecutions = (
```

Comment:
I suppose I prefer being more explicit and having it be very clear where all the "message parsing" functions are getting their output, rather than a generic helper like this where it's a bit less clear

Reply:
Not a hill I'll die on, but it feels like a call like `getMessagePdrStat(message, STAT)` isn't particularly less clear than, `getMessagePdrRunningSTAT(message)`, and it eliminates a bunch of boilerplate tests. 

That said, there isn't a concrete technical objection here as maintainability isn't a legit concern given the nature of these methods.